Unittests/MethodArguments/MethodArgumentFromInvocationNullAnnotatorTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) rubicon IT GmbH
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using NullableReferenceTypesRewriter.ConsoleApplication.MethodArguments;
+using NUnit.Framework;
+
+namespace NUnit2To3SyntaxConverter.Unittests.MethodArguments
+{
+  public class MethodArgumentFromInvocationNullAnnotatorTest
+  {
+    private const string c_template = "%TEMPLATE%";
+
+    private const string c_argumentTemplate =
+        @"public class ArgumentTest
+{
+  public void TestMethod (int structType, object referenceType1, double structType2, string referenceType2)
+  {
+  }
+
+  public void CallSite ()
+  {
+    %TEMPLATE%;
+  }
+}";
+
+    [Test]
+    public void MethodArgumentFromInvocationNullAnnotator_InvokesCallbackWithSingleNullableArgument ()
+    {
+      var source = c_argumentTemplate.Replace (c_template, "TestMethod (1, null, 2.3, new string())");

```

Comment:
IMO this is OK the way it is, but could also be written using `string.Format`, might make it easier to understand.